### PR TITLE
Private Chat

### PR DESCRIPTION
**Pull request by tech-savvy-guy**
Removed the condition of checking the `GROUPID` of the `messages`.

The Bot should (**hopefully**) now reply to private chats as well...

### CODE REVIEW

**Comment by barbax7** [comment]
this afternoon i'll give a look, thank you

**Comment by tech-savvy-guy** [comment]
Great! Let me know if something's wrong...

**Review by barbax7 — COMMENTED** [review]
I can't merge this code because autodelete function won't work in private chats
So we need new handler for private chat, or decorators, in order to choose when use autodelete and when not

File: main.py [hunk 1]
```diff
@@ -27,33 +27,31 @@
...
+    keys = db.select("SELECT Key, Description FROM Notes")
+    if keys:
+        keys.sort()
+        getkeytext = "Use '/get key' - to get note.\n\n<b>Keys:</b>\n"
+        for key in keys:
+            getkeytext+=f"<code>{str(key[0])}</code> - <i>{str(key[1])}</i>\n"
+        msg = bot.send_message(message.chat.id, getkeytext, reply_to_message_id = has_reply(message))
+    else:
+        msg = bot.send_message(message.chat.id, "There are no keys yet. Use\n'/add_note key description\ntext'")
+    bot.delete_message(message.chat.id, message.message_id)
```
Comment (barbax7) [inline main.py]:
```suggestion
    if message.chat.id == GROUPID: bot.delete_message(message.chat.id, message.message_id)
```

File: main.py [hunk 2]
```diff
@@ -27,33 +27,31 @@
...
+        if key:
+            bot.send_message(message.chat.id, str(key[0][0]), reply_to_message_id = has_reply(message))
         else:
-            msg = bot.send_message(message.chat.id, "Use /get key - to get note.")
-        bot.delete_message(message.chat.id, message.message_id)
-        autodelete(bot, msg)
+            msg = bot.send_message(message.chat.id, "Note with this key isn't available❌\n/notes")
+    else:
+        msg = bot.send_message(message.chat.id, "Use /get key - to get note.")
+    bot.delete_message(message.chat.id, message.message_id)
```
Comment (barbax7) [inline main.py]:
```suggestion
    if message.chat.id == GROUPID: bot.delete_message(message.chat.id, message.message_id)
```

File: main.py [hunk 3]
```diff
@@ -121,20 +125,21 @@ def add_admins(message: Message):
...
 
 @bot.message_handler(commands=['help'])
 def help_func(message):
-    if message.chat.id == GROUPID:
-        msg = bot.send_message(message.chat.id, "Usage:\n/notes - list of notes\n/add_note - add new note\n/delete_note - delete note\n/get - get note\n/add_admin - add new moderator")
-        autodelete(bot, msg, 2)
-        autodelete(bot, message, 0)
+    msg = bot.send_message(message.chat.id, "Usage:\n/notes - list of notes\n/add_note - add new note\n/delete_note - delete note\n/get - get note\n/add_admin - add new moderator")
+    autodelete(bot, msg, 2)
+    autodelete(bot, message, 0)
```
Comment (barbax7) [inline main.py]:
```suggestion
    if message.chat.id == GROUPID: autodelete(bot, message, 0)
```

File: main.py [hunk 4]
```diff
@@ -121,20 +125,21 @@ def add_admins(message: Message):
             msg = bot.send_message(message.chat.id, "You're not a moderator❌")
         autodelete(bot, msg, 2)
         autodelete(bot, message, 1)
+    else:
+        bot.send_message(message.chat.id, parse_mode="HTML", text="<b>Admins can be added ONLY in our group,\n\
```
Comment (barbax7) [inline main.py]: Every `parse_mode="HTML"` are useless because already defined in bot object

File: main.py [hunk 5]
```diff
@@ -121,20 +125,21 @@ def add_admins(message: Message):
...
 def welcome(message: Message):
-    if message.chat.id == GROUPID:
-        msg = bot.send_message(message.chat.id, "Hi, I'm online!\nSend /help")
-        autodelete(bot, msg, 0.5)
-    elif message.chat.type == 'private':
+    msg = bot.send_message(message.chat.id, "Hi, I'm online!\nSend /help")
+    autodelete(bot, msg, 0.5)
+    if message.chat.id != GROUPID:
         bot.send_message(message.chat.id, "Join pyTelegramBotApi's Telegram Group to get code snippets\n\nhttps://telegram.me/joinchat/Bn4ixj84FIZVkwhk2jag6A")
     autodelete(bot, message, 0)
```
Comment (barbax7) [inline main.py]:
```suggestion
    if message.chat.id == GROUPID:
        msg = bot.send_message(message.chat.id, "Hi, I'm online!\nSend /help")
        autodelete(bot, msg, 0.5)
        autodelete(bot, message, 0)
    else:
        bot.send_message(message.chat.id, "Join pyTelegramBotApi's Telegram Group to get code snippets\n\nhttps://telegram.me/joinchat/Bn4ixj84FIZVkwhk2jag6A")
```

**Review by barbax7 — CHANGES_REQUESTED** [review]
These are some examples. In this way it could work

File: main.py (hunk 3 above)
Comment (tech-savvy-guy) [reply main.py]: What about `autodelete(bot, msg, 2)` in `Line 135`?

File: main.py (hunk 4 above)
Comment (tech-savvy-guy) [reply main.py]: Oops, my bad :)

File: main.py (hunk 3 above)
Comment (barbax7) [reply main.py]: line 135 will delete bot's message, not user's message. so no issue

Comment (tech-savvy-guy) [reply main.py]: Oh, I see...

**Comment by tech-savvy-guy** [comment]
I have implemented all the changes :)

Hope it works now!

**Review by barbax7 — APPROVED** [review]

**Comment by barbax7** [comment]
I'll do some tests, but i think it will work, thank you!

**Pull request merged**